bounties/level-2/sep10-javascript-client.md

```diff
@@ -0,0 +1,68 @@
+# JavaScript SEP-0010 Client Reference Implementation
+
+## Award
+| Hunter | Reviewer | Type
+| :- | :- | :-
+| 1500 XLM | 1000 XLM | Capped (3)
```

Comment:
This will be pretty easy from a frontend perspective. It's just consuming an API endpoint, there isn't really much going on from a client side perspective. For this reason I'd say let's put this bounty at 1200 XLM Hunter and 800 XLM Reviewer.